covid19dp_submission/download_analyses.py

```diff
@@ -127,6 +133,20 @@ def download_files(analyses_array, download_target_dir, processed_analyses_file)
                     os.remove(download_file_path)
 
 
+def download_files_via_aspera(analyses_array, download_target_dir, processed_analyses_file, ascp, aspera_id_dsa,
+                              batch_size=100):
+    logger.info(f"total number of files to download: {len(analyses_array)}")
+    with open(processed_analyses_file, 'a') as open_file:
+        for analysis_batch in chunked(analyses_array, batch_size):
+            download_urls = [
+                f"era-fasp@{analysis['submitted_aspera']}" for analysis in analysis_batch if analysis
+            ]
+            command = f'{ascp} -i {aspera_id_dsa} -QT -l 300m -P 33001 {" ".join(download_urls)} {download_target_dir}'
```

Comment:
Suggest adding some info on the parameters used here..

Reply:
Add a retry mechanism. Not super happy about it though.